### PR TITLE
Feature/attempt to refine code

### PR DESCRIPTION
**Pull request by rpoletaev**

### CODE REVIEW

File: packages/api/login.go [hunk 1]
```diff
@@ -153,24 +153,23 @@ func login(w http.ResponseWriter, r *http.Request, data *apiData, logger *log.En
 	if r, ok := data.params["role_id"]; ok {
 		role := r.(int64)
 		if role > 0 {
-			ok, err := model.MemberHasRole(nil, ecosystemID, wallet, role)
-			if err != nil {
-				logger.WithFields(log.Fields{
+			getRoleErrorFields := func() log.Fields {
```
Comment (potehinre) [inline packages/api/login.go]: you could do requestLogger := logger.WithFields(log.Fields{...}} instead of that

File: packages/api/login.go [hunk 2]
```diff
@@ -153,24 +153,23 @@ func login(w http.ResponseWriter, r *http.Request, data *apiData, logger *log.En
...
 					"type":      consts.DBError,
 					"member":    wallet,
 					"role":      role,
-					"ecosystem": ecosystemID}).Error("check role")
+					"ecosystem": ecosystemID}
+			}
+
+			ok, err := model.MemberHasRole(nil, ecosystemID, wallet, role)
+			if err != nil {
+				logger.WithFields(getRoleErrorFields()).Error("check role")
```
Comment (potehinre) [inline packages/api/login.go]: requestLogger.Error("check role")

File: packages/api/login.go [hunk 3]
```diff
@@ -153,24 +153,23 @@ func login(w http.ResponseWriter, r *http.Request, data *apiData, logger *log.En
...
 			}
 
 			if !ok {
-				logger.WithFields(log.Fields{
-					"type":      consts.NotFound,
-					"member":    wallet,
-					"role":      role,
-					"ecosystem": ecosystemID,
-				}).Error("member hasn't role")
+				logger.WithFields(getRoleErrorFields()).Error("member hasn't role")
```
Comment (potehinre) [inline packages/api/login.go]: requestLogger.Error("member hasnt role")

File: packages/api/refresh.go [hunk 4]
```diff
@@ -83,14 +83,16 @@ func getRefreshTokenClaims(w http.ResponseWriter, data *apiData, logger *log.Ent
 		logger.WithFields(log.Fields{"type": consts.JWTError}).Error("getting jwt claims")
 		return nil, errorAPI(w, `E_TOKEN`, http.StatusBadRequest)
 	}
-	token, err := jwt.ParseWithClaims(data.params[`token`].(string), &JWTClaims{},
```
Comment (potehinre) [inline packages/api/refresh.go:86]: that one was better

**Review by potehinre — CHANGES_REQUESTED** [review]

File: packages/api/refresh.go (hunk 4 above)
Comment (rpoletaev) [reply packages/api/refresh.go:86]: call of jwt.ParseWithClaims more clear when he is inline

**Review by potehinre — APPROVED** [review]

**Pull request closed without merging**